Maquetado/scss/mixins.scss

```
@@ -0,0 +1,18 @@
+
+
+@mixin bookDetailfontStyle {
+  color: $black;
+  display: block;
+  font-family: Roboto;
+  font-weight: 700;
+}
+
+@mixin flexBox {
+  align-items: center;
+  display: flex;
+  justify-content: center;  
+}
+@mixin buttonsStyle {
+border-radius: 5px;
+height: 12%;
+}
```

Check the indentation here
```suggestion
@mixin buttonsStyle {
  border-radius: 5px;
  height: 12%;
}
```